providers/multiprovider/README.md

```diff
@@ -0,0 +1,97 @@
+# OpenFeature Multi-Provider for Java
```

Comment:
Nice work on the README.

Small nit: 1 sentence per line makes reviewing and change-deltas much nicer.

Reply:
Thanks, updated.